tools/pip_install.sh

```diff
@@ -89,4 +89,5 @@ ontology_validator_setup
 instance_validator_setup
 explorer_setup
 guid_generator_setup
+abel_setup
```

Comment:
does this add the pip in the abel setup?

Reply:
Yeah I don't think anymore needs to be added. I checked in the instance validator, for example, and there aren't extra files besides `setup.py`. `python -m pip install .` is alaready defined in the `abel_setup()` function.